rspace/src/test/scala/coop/rchain/rspace/HotStoreSpec.scala

```diff
@@ -1377,31 +1130,25 @@ trait InMemHotStoreSpec extends HotStoreSpec[Task, Task.Par] {
       ) => F[Unit]
   ) =
     (for {
-      historyState <- Ref.of[F, HotStoreState[String, Pattern, String, StringsCaptor]](
-                       HotStoreState[String, Pattern, String, StringsCaptor]()
-                     )
+      historyState <- Ref[F].of(HotStoreState[String, Pattern, String, StringsCaptor]())
       history = {
-        implicit val hs = historyState
-        new History[F, String, Pattern, String, StringsCaptor]
+        new History[F, String, Pattern, String, StringsCaptor](historyState)
```

Comment:
Curly braces are not necessary now.

Reply:
@tgrospic, [done](https://github.com/rchain/rchain/pull/3557/commits/a395749d5017a6e535a7eccc129fb72bf38e69d0).